storage.bs

```diff
@@ -178,7 +178,21 @@ larger <a>site storage quota</a>. Factors such as navigation frequency, recency
 bookmarking, and <a href="#persistence">permission</a> for {{"persistent-storage"}} can be used as
 indications of "popularity".
 
+The <dfn export>application cache site storage usage</dfn> for an <a for=/>origin</a>
+<var>origin</var> is a rough estimate of the amount of bytes used in <a>Application Cache</a>
+in <var>origin</var>'s <a>site storage unit</a>. [[!HTML]]
 
+The <dfn export>caches site storage usage</dfn> for an <a for=/>origin</a>
+<var>origin</var> is a rough estimate of the amount of bytes used in <a attribute>caches</a> API
```

Comment:
This should be something like `{{Caches}}` or `{{Cache}}`, no? Referring to the getter seems a little weird.

Reply:
My understanding was that it should link here: https://w3c.github.io/ServiceWorker/#cache-objects

When I used `{{Cache}}`, it linked here https://w3c.github.io/ServiceWorker/#cache.

When I used `{{Caches}}` or `{{cache-objects}}`, it did not create a link (it was just grey text).

Please advise.

Reply:
I guess you want `{{CacheStorage}}`? @jakearchibald thoughts?

Reply:
The other definitions don't link to an interface... I'd expect a link to the top level _concept_ rather than an interface, e.g. https://w3c.github.io/ServiceWorker/#cache-objects 

I don't think there's an export from SW there. So yeah, @jakearchibald ?

Reply:
Note that we could do this by adding a `<pre class=anchors>...</pre>` section w/o an explicit export from SW, but let's figure out what we want here. FWIW over in https://wicg.github.io/web-locks/ I wanted to do something similar and ended up with "Caches [Service-Workers])" where "Caches" was not a link and latter bit was just a biblio link, assuming readers would figure it out.